commands/motor/MotorPositionSet.java

```diff
@@ -69,6 +67,9 @@ protected void execute() {
 	}
 	
 	@Override
+	protected void execute() {}
```

Comment:
Yes.
TL;DR: It would be problematic for the motion controller to silently fail; the fallbackCommand should be started on sensor failure, regardless of whether it was during initialization or while running.
Given that the way some errors are detected is through timeouts, it is actually more likely that an error will become apparent while running than during initialization. While the motion controller will disable itself either way, it does not have any ability to trigger the fallback without the command actively checking for errors. It might be possible to implement that, but I am inclined to prefer to separate the fallback command from the controller.